tests/step_implementers/create_container_image/test_buildah.py

```diff
@@ -103,42 +126,72 @@ def test_create_container_image_specify_buildah_implementer_with_destination_inv
                 run_step_test_with_result_validation(temp_dir, 'create-container-image', config, {})
 
     @patch('sh.buildah', create=True)
-    def test_create_container_image_specify_buildah_implementer_with_destination_valid_dockerfile(self, buildah_mock):
+    def test_create_container_image_specify_buildah_implementer_with_valid_dockerfile(self, buildah_mock):
         with TempDirectory() as temp_dir:
-            destination = 'localhost'
+            application_name = 'foo'
+            service_name = 'bar'
+            version = 'latest'
+            tag = 'localhost/{application_name}/{service_name}:{version}'.format(
+                application_name=application_name,
+                service_name=service_name,
+                version=version
+            )
+            image_tar_file = 'image-{application_name}-{service_name}-{version}.tar'.format(
+                application_name=application_name,
+                service_name=service_name,
+                version=version
+            )
             file = 'Dockerfile'
             temp_dir.write(file,b'FROM registry.access.redhat.com/ubi8:latest')
             config = {
                 'tssc-config': {
+                    'global-defaults': {
+                        'application-name': application_name,
+                        'service-name': service_name
+                    },
                     'create-container-image': {
                         'implementer': 'Buildah',
                         'config': {
-                            'destination' : destination,
                             'context' : temp_dir.path
                         }
                     }
                 }
             }
 
-            image_tar_file = 'image-latest.tar'
-            expected_step_results = {'tssc-results': {'create-container-image': {'image-tag': destination + ':latest', 'image-tar-file': image_tar_file}}}
+            expected_step_results = {'tssc-results': {
+                'create-container-image': {
+                    'image-tag': tag,
+                    'image-tar-file': image_tar_file
+                }
+            }}
             run_step_test_with_result_validation(temp_dir, 'create-container-image', config, expected_step_results)
             buildah_mock.bud.assert_called_once_with(
                 '--format=oci',
                 '--tls-verify=true',
                 '--layers',
                 '-f', file,
-                '-t', destination + ':latest',
+                '-t', tag,
```

Comment:
its cuz `destination` never got removed from the `buildah` step implementer and ended up "having" to fix it because it was either that or propigate bad tests when fixing the tests for the changes I wanted.